### ISSUE TITLE
Static Stage Training Setup

### ISSUE THREAD

**Issue by AtaAtasoy** [WU-CVGL/DreamMesh4D#4]
Hi, thank you for this great work. I was wondering how I could replicate the static training mentioned in the paper. The paper mentions:

> In the process of both coarse mesh generation and SuGaR refinement, the loss in Equation 3 is utilized for supervision, with the strength of different terms as λ_SDS = 0.1, λ_ref = 1000 and λ_mask = 500. For the generation of coarse mesh, a set of randomly initialized 3D Gaussians are optimized for 3000 steps in total. In the former 1500 steps, we do densification and pruning every 100 steps. After the 1500 steps, densification and pruning are stopped, and we introduce additional opacity binary and density regularization terms as described in  into optimization until step 3000. Finally, we prune all Gaussians with opacity less than 0.5 and extract coarse mesh through Poisson reconstruction. Afterwards, we bind x = 6 new flat Gaussians to each triangle face of the coarse mesh, and do optimization for 2000 steps.

This makes me feel like just running the static stage as in the readme doesn't replicate this training setup. 

I would appreciate any sort of feedback regarding this. Thanks!

**Comment by lizhiqi49** [comment]
While the core contribution of our work is the dynamic optimization from a static mesh, for the static mesh generation, we simply opt for NeRF/NeuS-based mesh generation pipeline, which is much more straightforward than the method we used in our main paper, and can achieve similar results. We will also replenish the Gaussian-SuGaR static stage optimization pipeline later for better alignment with our paper.
